posts/app.py

```diff
@@ -60,28 +61,52 @@ def upload_new_post():
 @app.route('/v1/', methods=['GET'])
 def get_all_posts():
     """Get all posts for the whole event."""
-    # TODO(mukobi) change from fake data to real query from db
-    return jsonify({
```

Comment:
Nice! glad to be rid of this fake static data

Reply:
There goes our fake reliability